pkgs/by-name/fo/formatjson5/package.nix

```diff
@@ -19,6 +20,13 @@ rustPlatform.buildRustPackage rec {
     hash = "sha256-Lredw/Fez+2U2++ShZcKTFCv8Qpai9YUvqvpGjG5W0o=";
   };
 
+  patches = [
+    (fetchpatch {
```

Comment:
nit: I like having a comment about a patch so you don't have to visit the url to see what the patch does, but not strictly required. ;)

Reply:
Oh that's a good point. I'll do that next time!

Thanks for the review and merge